css/main.css

```diff
@@ -51,6 +52,42 @@ header {
   color: white;
 }
 
+#mobile-nav {
+  background-color: #3c3a39;
+  display: none;
+  height: 100vh;
+  position: fixed;
+  inset: 0 0 0 0;
+  z-index: 10;
+}
+
+#mobile-nav > ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: start;
+  margin: 0 1rem;
+}
+
+.menu-close {
+  color: #67798e;
+  position: absolute;
+  right: 0;
+  margin: 0 1rem;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Thank you for implementing the changes but you need to work again in the mobile menu layout section I see your design is still different from the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly follow the Figma design to make it look perfect and get the most professional results.


![2022-05-21_00-11-57](https://user-images.githubusercontent.com/63915024/169597918-1d43ce98-04c1-4c82-ad8f-a64b6f0339d7.png)